tests/contrib/django_util/settings.py

```diff
@@ -12,8 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""This module provides a base Django settings module used by the rest of the
+tests."""
+
 import os
 
+INSTALLED_APPS = [
```

Comment:
Never write a list where a tuple will do, especially for the value of a constant. Can this be a tuple? Or does something that uses it check that it's a list, forcing everything that uses it to be extra careful not to accidentally mutate it?